qml/MKSConnectBtn.qml

```diff
@@ -59,7 +59,7 @@ Item {
         Button {
             height: UM.Theme.getSize("save_button_save_to_button").height;
             onClicked: Cura.MachineManager.printerOutputDevices[0].requestAuthentication();
-            style: UM.Theme.styles.print_setup_action_button;
+            // style: UM.Theme.styles.print_setup_action_button;
```

Comment:
The style property seems to be deprecated. I'm not sure what should be done for styling (I do know Cura.PrimaryButton will make the button blue, and SecondaryButton makes is grayish/white). I've just commented out the styles for now until the correct solution is decided.

Reply:
Yeah, I think PrimaryButton styles will be correct as default style.